point.py

```diff
@@ -0,0 +1,135 @@
+import math  # I am guessing that you will need to use the math module
+import json  # I would like you to use the JSON module for reading geojson (for now)
+from math import sqrt
+from . import utils
+
+class Point():
+    def __init__(self, x=0, y=0, mark=[]):
+        self.x = x
+        self.y = y
+        self.mark = mark
+        
+    def read_geojson(self,input_file):
+        with open(input_file, 'r') as f:
+            gj=json.load(f)
+            
+            # Please use the python json module (imported above)
+            # to solve this one.
+            
+            return gj
+
+    def check_if_coincident(self,secondPoint):
+        return utils.check_coincident((self.x,self.y),secondPoint)

```

Comment:
How you used `utils.check_coincident` is precisely how you could path the read_geojson method in.